runtime/src/accounts_db.rs

```diff
@@ -189,6 +189,52 @@ impl ZeroLamport for AccountInfo {
     }
 }
 
+struct MultiThreadProgress<'a> {
+    last_update: Instant,
+    my_last_report_count: u64,
+    total_count: &'a AtomicU64,
+    report_delay_secs: u64,
```

Comment:
Nitpick - what you do you think about `report_interval_secs` ?

Reply:
good suggestion. Hopefully I fix that when this code next changes.